app/models/mediaflux/project_report.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module Mediaflux
+  class ProjectReport < Request
+    # Specifies the Mediaflux service to use when creating assets
```

Comment:
The comment is wrong, right? Could we remove it?

Reply:
Thanks for the catch @hectorcorrea!  I went ahead and updated the comment.  Copy 🍝 error